cli/targets/static.js

```diff
@@ -354,8 +360,66 @@ function toJsType(field) {
     return type;
 }
 
+function syntaxForType(type) {
```

Comment:
Nit, please add curly brackets around if/else for readability

Reply:
Done, changes pushed